### PR TITLE
Add docker setup for print v2

### LINKED ISSUE #154: Dockerize mapfish-print version 2 environment
At the moment only mapfish3 environment is based on docker, the docker should be set up for the version 2 as well.

### PR DESCRIPTION
**Pull request by dnlkoch**
Resolves #154.

### CODE REVIEW

**Comment by annarieger** [comment]
@dnlkoch, I resolved remaining issues and finalized this PR. Would be nice, if you could take a look someday :)

File: docker/mapfish-print-v2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM terrestris/tomcat:8.5.37
+
+RUN apk -U upgrade --update && \
+    apk add curl && \
+    rm -rf $CATALINA_HOME/webapps/*
+
+RUN curl -jkSL -o /tmp/print.war http://central.maven.org/maven2/org/mapfish/print/print-servlet/2.1.4/print-servlet-2.1.4.war && \
```
Comment (dnlkoch) [inline docker/mapfish-print-v2/Dockerfile]: Please switch to `https` (e.g. by using `https://repo.maven.apache.org/maven2/org/mapfish/print/print-servlet/`) and make version configurable.

File: docker/mapfish-print-v2/print-config/config.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    # override the default outputFileName defined above
+#    outputFilename: A4_Layout
+    mainPage:
+      rotation: true
+      pageSize: A4
+      header:
+        height: 50
+        items:
+          - !text
+            font: Helvetica
```
Comment (dnlkoch) [inline docker/mapfish-print-v2/print-config/config.yaml]: I think this wont be available on the container, please change to `DejaVu Sans` or similar.

File: webpack.development.config.js [hunk 3]
```diff
@@ -8,10 +8,49 @@ module.exports = merge(commonConfig, {
...
+      '/print-v3/**': {
+        target: {
+          host: '127.0.0.1',
+          port: 18083
+        },
+        pathRewrite: {
+          '^/print-v3': '/print'
+        }
+      },
+      '/print/pdf/**': {
```
Comment (dnlkoch) [inline webpack.development.config.js]: Why is this needed? Shouldn't it be `/print-v2/pdf/**`?

File: docker/mapfish-print-v2/Dockerfile (hunk 1 above)
Comment (annarieger) [reply docker/mapfish-print-v2/Dockerfile]: Changed and configured in 1e751c0

File: docker/mapfish-print-v2/print-config/config.yaml (hunk 2 above)
Comment (annarieger) [reply docker/mapfish-print-v2/print-config/config.yaml]: Switched to `arial` in cdc8c05

File: webpack.development.config.js (hunk 3 above)
Comment (annarieger) [reply webpack.development.config.js]: This is needed to be able to redirect to the correct print download URL. If called via example (from localhost:9000), the URLs in `info.json` response looks like this:
```
{
  ...
  "printURL": "http://localhost:9000/print/pdf/print.pdf",
  "createURL": "http://localhost:9000/print/pdf/create.json"
}
```
With this additional proxies we ensure that `http://localhost:9000/print/pdf/` will be redirect to `http://localhost:18082/print/pdf/`

The same applies to the next two entries (`/print/status/` and `/print/report/`) in proxy array and is needed for redirects while operating with print servlet v3
